plugins/catalog/src/components/useOwnUser.ts

```diff
@@ -30,12 +30,12 @@ export function useOwnUser(): AsyncState<UserEntity | undefined> {
   // TODO: get the full entity (or at least the full entity name) from the
   // identityApi
   return useAsync(
-    () =>
-      catalogApi.getEntityByName({
+    async () =>
+      await catalogApi.getEntityByName({
```

Comment:
Had to use `async/await` here, otherwise the hook would blow up the catalog page.